js/ui/components/clubhouse.js

```diff
@@ -137,6 +137,8 @@ var ClubhouseButtonManager = new Lang.Class({
         this._clubhouseWindowActor = null;
         this._clubhouseNotifyHandler = 0;
 
+        this._visible = true;
```

Comment:
I find it a bit confusing that this flag is called `this._visible`, because `this._visible = true` does not imply that the button is actually visible (there are the other conditions). I think the meaning of this option would be clearer if it was called `this._enabled`.

Reply:
On second thought, I am just going to merge this as-is.